README.md

```diff
@@ -24,3 +33,96 @@ For the documentation of the module in `README.md`, the following guidelines app
 
 For the short module description (`shortdesc:`), the following guidelines apply:
 - The description should start with a phrase like "The glorb module reticulates splines" or "The tree module can be used to plant trees".
+
+### Local Module Config
```

Comment:
This section should also:
- Make the purpose of the configuration as storing data for the module to use when running post-boot clearer.
- Provide some tips and example pieces of code for the implementation.

Reply:
1. I think that this makes it clearer
[`37c6433` (#175)](https://github.com/blue-build/modules/pull/175/commits/37c6433320c1af378b5e3a989d0537e67a1a7c0f)
2. I will think about what is the standardized piece of code to present, which would work for every module of this type. I'm open for suggestions.